### ISSUE TITLE
Individual images can be deleted from upload history

### ISSUE THREAD

**Issue by neel1998** [fossasia/phimpme-android#2602]
**Describe the feature you'd like**

Right now user can only delete all the images from upload history, there should be option to select individual images and delete them

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

**Comment by yashk2000** [comment]
If you open an image, you can delete it from upload history. There is a delete option at the bottom.

**Comment by neel1998** [comment]
yes. but if i want to delete multiple images and not all, being able to select them by long press and then deleting in one go would certainly enhance user experience

**Comment by yashk2000** [comment]
Yes. That should be done. I did not get it because the issue heading is misleading. Please give appropriate issue headings.
